src/redist/targets/packaging/windows/dotnetCoreUninstall/dotnet.wxs

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?include "Variables.wxi" ?>
+  <Product Id="*" Name="$(var.ProductName)" Language="$(var.ProductLanguage)" Version="$(var.ProductVersion)" Manufacturer="$(var.Manufacturer)" UpgradeCode="$(var.UpgradeCode)">
+    <Package Compressed="yes" InstallScope="perMachine" InstallerVersion="200" />
+
+    <MajorUpgrade DowngradeErrorMessage="$(var.DowngradeErrorMessage)" Schedule="afterInstallInitialize"/>
+
+    <?if $(var.Platform)=x86?>
```

Comment:
Are you building 2 installers - one for x86 and one for x64?

Reply:
no, but i don't want to remove 64 part. Since i could if I add a new leg

Reply:
Sounds good to me.